src/BinaryParsers/BinaryParsers.csproj

```diff
@@ -44,7 +44,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="Microsoft.CodeAnalysis.NetAnalyzers" Version="7.0.0">
```

Comment:
>7.0.0 [](http://example.com/codeflow?start=76&length=5)

I also see we have 6.0.0
should we upgrade them as well

Reply:
Yes, what made sense is to update ONLY the build.common.props and to remove ALL the other NetAnalyzer references as they are just overriding what is in build.common.props.

Now with build.common.props updated to 7.0.1 then everything will use that.